Dockerfile

```diff
@@ -2,18 +2,26 @@ FROM node:latest AS build-env
 WORKDIR /app
 
 # Prepare the dependencies
-COPY package*.json /app/
-RUN npm install && npm cache clean --force
+COPY package.json \
+     yarn.lock \
+     .yarnrc.yml \
+     /app/
+COPY .yarn /app/.yarn
+RUN corepack enable && yarn
 # Copy everything else
 COPY . /app
 ARG NODE_ENV
 ENV NODE_ENV $NODE_ENV
-RUN npm run build
+# workaround for error:0308010C:digital envelope routines::unsupported
+ENV NODE_OPTIONS --openssl-legacy-provider
```

Comment:
AFIK, if we decided to use node v16, `--openssl-legacy-provider`may not be needed.  ([see also](https://nodejs.org/en/blog/release/v17.0.0/#openssl-3-0)).

Reply:
👍 It might have been one of the legacy libraries that required this fix, it seems like it is no longer required to build the project (although we decided to remove Dockerfile 😅)